public/controllers/management/components/upload-files.js

```diff
@@ -83,26 +90,38 @@ export class UploadFiles extends Component {
             clearInterval(interval);
             if (files.length === this.state.files.length) {
               try {
-                await this.props.upload(files, this.props.resource);
-                this.closePopover();
-                this.props.onSuccess && this.props.onSuccess(files);
-                this.showToast(
-                  'success',
-                  'Success',
-                  <Fragment>
-                    <div>Successfully imported</div>
-                    <ul>
-                      {files.map(f => (
-                        <li
-                          key={`ruleset-imported-file-${f.file}`}
-                          style={{ listStyle: 'circle' }}
-                        >
-                          {f.file}
-                        </li>
-                      ))}
-                    </ul>
-                  </Fragment>
+                const noUploaded = [];
```

Comment:
What does `noUpload` stands for? Files not uploaded due to error?

If so, rename the variable to something more descriptive, as `failedUploads` or `notUploadedFiles`.

Reply:
Solved